models/sql-utils.go

```diff
@@ -34,3 +40,32 @@ func sanitizeOrderInput(order string, validColumns []string) string {
 
 	return ""
 }
+
+var (
+	dbHandler database.Handler
+)
+
+func SetNewDBInstance() {
```

Comment:
@MUzairS15  Can you make this function thread safe?

Reply:
@Revolyssup Is it ok now??

Reply:
Since you have exported SetNewDBInstance, it would be good to have thread safety at that level. So just move the Locking and unlocking of mutexes inside of that function.

Reply:
Oh ok, inside `SetNewDBInstance` function.